bundles/org.openhab.ui/web/src/components/widgets/system/oh-input.vue

```diff
@@ -21,37 +17,65 @@
 </style>
 
 <script>
+import dayjs from 'dayjs'
+
 import mixin from '../widget-mixin'
 import { OhInputDefinition } from '@/assets/definitions/widgets/system'
 
 export default {
   mixins: [mixin],
   widget: OhInputDefinition,
-  mounted () {
-    delete this.config.value
+  data () {
+    return {
+      pendingUpdate: null
+    }
```

Comment:
Thanks forr this. I was preparing a PR as the context value should indeed not be used.